xcube/server/api.py

```diff
@@ -632,10 +659,21 @@ class ApiError(Exception):
     """
 
     def __init__(self,
-                 status_code: int,
+                 status_code: int = 500,
```

Comment:
Seems a bit random to set 500 as default. I don't see a good reason for this.

Reply:
By accident! I'll fix that.

Reply:
Fixed.